lib/rubycritic/rake_task.rb

```diff
@@ -0,0 +1,50 @@
+require "rake"
+require "rake/tasklib"
+
+module Rubycritic
+  class RakeTask < ::Rake::TaskLib
+    attr_accessor :name
+    attr_accessor :report
+    attr_accessor :format
+    attr_accessor :paths
+    attr_accessor :options
+
+    def initialize(*args, &task_block)
+      @name = args.shift || :rubycritic
+      @fail_on_error = true
+      @options = []
+      @format = "html"
+      @report = nil
+      @paths = []
+
+      desc "Run RubyCritic" unless ::Rake.application.last_comment
+
+      task(name, *args) do |_, task_args|
```

Comment:
Bugger! I had changed that, but when I messed up the rebase, I apparently managed to merge the wrong thing. I had copied what you do in reek...

I'll fix it!